contracts/src/core/EigenDAServiceManager.sol

```diff
@@ -182,8 +188,68 @@ contract EigenDAServiceManager is Initializable, OwnableUpgradeable, EigenDAServ
         return uint32(block.number) + STORE_DURATION_BLOCKS + BLOCK_STALE_MEASURE;
     }
 
-    /// @dev need to override function here since its defined in both these contracts
-    function owner() public view override(OwnableUpgradeable, IServiceManager) returns (address) {
-        return OwnableUpgradeable.owner();
+    /**
+     * @notice Returns the list of strategies that the operator has potentially restaked on the AVS
+     * @param operator The address of the operator to get restaked strategies for
+     * @dev This function is intended to be called off-chain
+     * @dev No guarantee is made on whether the operator has shares for a strategy in a quorum or uniqueness 
+     *      of each element in the returned array. The off-chain service should do that validation separately
+     */
+    function getOperatorRestakedStrategies(address operator) external view returns (address[] memory) {
```

Comment:
The implementations in these methods are not the most recent implementations in `ServiceManagerBase`. Do we want to use those, or were these changes made for a reason?

Reply:
Is it possible to just inherit from `ServiceManagerBase`?